README.md

```diff
@@ -0,0 +1,2 @@
+# merkle-trie [![Build Status](https://travis-ci.org/CodeChain-io/rust-merkle-trie.svg?branch=master)](https://travis-ci.org/CodeChain-io/rust-merkle-trie) [![License: AGPL v3](https://img.shields.io/badge/License-AGPL%20v3-blue.svg)](https://www.gnu.org/licenses/agpl-3.0)
```

Comment:
We don't use Travis for this library. We are using GitHub Actions.

Reply:
@sgkim126  I want to know too about which license to use.

Reply:
Currently, it uses AGPL.

Reply:
Are we the only ones who use this library? I think it is acceptable to use a low-level license if it is available to everyone. I want to know why we use AGPL. Could you explain it? : )

Reply:
It's because it uses the libraries that are AGPL.
And the reason that the dependant libraries are AGPL is that the libraries used the source of GPLv3 and AGPL.

Reply:
I see! Thank you : )) I modified it.